src/utils/helpers/structure/generateConsolidatorResponsesProgress.ts

```diff
@@ -56,8 +56,8 @@ const generatePageConsolidationProgress = (page: Page, assignedSections: string[
   if (!assignedSections.includes(page.sectionCode)) page.consolidationProgress = initial
   else {
     const totalReviewable = page.state.filter(
-      ({ isAssigned, thisReviewLatestResponse, element }) =>
-        (isAssigned || !!thisReviewLatestResponse) && element.isVisible
+      ({ isAssigned, isActiveReviewResponse, thisReviewLatestResponse, element }) => 
+        ((isAssigned && isActiveReviewResponse) || !!thisReviewLatestResponse) && element.isVisible
```

Comment:
Oh yeah, I think I can see the problem -- you're using `isActiveReviewResponse` but on the Assignment page, there are no active review responses yet (cos it's not started).

2 options:
- fix this logic to not use `isActiveReviewResponse` but still fix the problem
- use a different method to determine reviewer action on Assignment page

Thoughts?

Reply:
I've just realised that `assignmentGroupedLevel` that is passed to the assignment rows has this info in it:
```
{
  "2": [
    {
      "id": 26,
      "reviewer": {
        "__typename": "User",
        "id": 5,
        "firstName": "Jessica",
        "lastName": "Jago"
      },
      "level": 2,
      "current": {
        "stage": {
          "id": 153,
          "name": "Avaliação técnica",
          "number": 2,
          "colour": "#E17E48"
        },
        "assignmentStatus": "ASSIGNED",
        "timeStageCreated": "2022-04-28T16:21:43.735+12:00",
        "timeStatusUpdated": "2022-04-28T16:22:52.765333+12:00"
      },
      "isCurrentUserReviewer": true,
      "isCurrentUserAssigner": false,
      "isFinalDecision": false,
      "isLastLevel": true,
      "isSelfAssignable": true,
      "isLocked": false,
      "allowedSections": [],
      "assignedSections": [
        "M1",
        "M2",
        "M3",
        "M4"
      ],
      "review": null
    }
  ]
}
```

So we should be able to use ` "isCurrentUserReviewer": true,` to determine the action rather than relying on `generateConsolidatorResponsesProgress.ts`, do you think?

Reply:
Also -- just wanted to point out that even without this change, the "Start" link only appears for sections that have questions answered:
<img width="946" alt="Screen Shot 2022-04-29 at 11 03 18 AM" src="https://user-images.githubusercontent.com/5456533/165861687-7bd5c3ea-d8c8-4914-824c-a1914f361e2e.png">

Reply:
This fix is working great now. Have fixed the other problem it caused -- see below.